File: app/src/main/java/rogerio/com/fractal/fractal.java
package rogerio.com.fractal;

import android.app.Activity;
import android.os.Bundle;

public class fractal extends Activity {
    /** Called when the activity is first created. */
	Screen screen; 
    @Override
    public void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        screen = new Screen(this);
        setContentView(screen);
    }
}

File: app/src/main/java/rogerio/com/fractal/fractal.kt
package rogerio.com.fractal

import android.app.Activity
import android.os.Bundle

class fractal : Activity() {
    /** Called when the activity is first created.  */
    var screen: Screen? = null
    public override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        screen = Screen(this)
        setContentView(screen)
    }
}